README.rst

```diff
@@ -63,8 +63,28 @@ Specifying an environment
 
 By default all accounts will be created on the 'stage' environment. You can set
 the ``FXA_ENV`` environment variable to target 'production' or 'stable'.
-Alternatively, you can override the ``fxa_urls`` fixture to specify the URL for
-you environment:
+
+If you need to override the environment for a subset of tests, or run tests against multiple environments, you can use the ``fxa_env`` marker:
+
+.. code-block:: python
+
+  @pytest.mark.fxa_env('production')
+  def test_production(fxa_account):
+      selenium.get('https://example.com/')
+      selenium.find_element(By.ID, 'email').send_keys(fxa_account.email)
+      selenium.find_element(By.ID, 'password').send_keys(fxa_account.password)
+      selenium.find_element(By.ID, 'login').click()
+
+
+  @pytest.mark.fxa_env('stage', 'stable')
+  def test_stage(fxa_account):
+      selenium.get('https://elpmaxe.com/')
```

Comment:
It doesn't really make a difference, but is this supposed to be ``https://example.com/``? 🤔

Reply:
That would be an impressive typo! Mozilla has a tendancy to register hostnames for staging environments that are transposed versions of the production hosts. For example mozilla.org becomes allizom.org... this is playfully doing the same with example.com.

Reply:
Haha got it! 👍